torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -702,31 +711,84 @@ def kernel_call():
 
         return benchmarker.benchmark_gpu(kernel_call, rep=40)
 
-    def clone_args(self, *args, **kwargs) -> Tuple[List[Any], Dict[str, Any]]:
-        from ..compile_fx import clone_preserve_strides
+    def copy_args_to_cpu_if_needed(self, *args, **kwargs):
```

Comment:
We still need to fix `benchmark_fused_nodes` so that we consistently do the same behavior. Can we patch `is_inference` and `is_backward` (or some other way, up to you) so that we always copy arguments inside benchmarking ? see, [here](https://github.com/pytorch/pytorch/blob/0878739b11f95fe092bffc0195d0e7a348612806/torch/_inductor/codegen/triton.py#L3180-L3182)

Reply:
Yeah, we should already be good there if I understand the ask. `benchmark_combo_kernel` and `benchmark_fused_nodes` call `clone_args` directly and I kept the behavior the same there (clone all mutated buffers -- no copying to cpu).